default/stringtables/en.txt

```diff
@@ -9710,6 +9717,18 @@ Psychic Fatigue
 CONC_CAMP_SLAVE_SPECIAL_DESC
 There may no longer be [[buildingtype BLD_CONC_CAMP]] operating on this planet today, but memories are still close to the surface, and remnants such as poorly hidden mass graves are still occasionally found. A psychic wound still lies upon the populace, and they do not thrive.
 
+RACIAL_PURITY_SENTINEL_SPECIAL
+Racial Purity
+
+RACIAL_PURITY_SENTINEL_SPECIAL_DESC
+This empire applies the [[policy PLC_RACIAL_PURITY]] policy.
+
+RACIAL_PURITY_REMNANT_SPECIAL
+Racial Resentment
+
+RACIAL_PURITY_REMNANT_SPECIAL_DESC
+This empire may not apply the [[policy PLC_RACIAL_PURITY]] policy anymore, but it has been operating relentlessly and memories are still close to the surface. The toll on [[metertype METER_HAPPINESS]] on the planets of the empire will need some time to fade out.
```

Comment:
```suggestion
This empire may not apply the [[policy PLC_RACIAL_PURITY]] policy soon after de-adeopting it, but memories of its adoption are still close to the surface. The toll on [[metertype METER_HAPPINESS]] on the planets of the empire will fade over time.
```

Reply:
I meant something like "Despite this empire no longer applies [[policy PLC_RACIAL_PURITY]] policy, it has been operating relentlessly and memories are still close to the surface. The toll on [[metertype METER_HAPPINESS]] on the planets of the empire will fade over time."
From the wording in your suggestion I understand "This empire can not adopt Racial Purity for a while", which is not what I meant. Am I misreading your wording?

Reply:
```suggestion
This empre recented de-adopted the [[policy PLC_RACIAL_PURITY]] policy and memories of its adoption linger in society. Its toll on [[metertype METER_HAPPINESS]] on the planets of the empire will fade over time.
```

Reply:
"recented" -> "recently" ??